README.md

```diff
@@ -2,58 +2,21 @@
 
```

Comment:
Remember to have consider on every project, your General Requirements this is important to do this flow because it's a standard to have nice projects and professional.

- Add a description about your features implemented on this branch on your Pull Request:
![Screen Shot 2021-09-21 at 3 48 23](https://user-images.githubusercontent.com/22258274/134141059-e1352f58-0d79-4353-9454-43dec487ae7b.png)

**Keep doing commits from local to GH repo in every change that you implement.**
- Kindly use descriptive commit messages - each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

<img width="822" alt="descriptive_commits_example" src="https://user-images.githubusercontent.com/22258274/134141428-b98fa291-324b-4a06-affc-b6186e009d47.png">


**If you require more information about these requested changes, kindly check your Project Requirements > General Requirements > Make sure that you documented your** [work in a professional way.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)